lib/net.js

```diff
@@ -1563,12 +1563,12 @@ exports.isIP = cares.isIP;
 
 
 exports.isIPv4 = function(input) {
-  return exports.isIP(input) === 4;
+  return cares.isIPv4(input);
```

Comment:
No particular reason, but I've just added check for symbol in js, so wrapper is necessary